models/tridimensional/docking_validation/dock_variants.py

```diff
@@ -122,12 +134,16 @@ def dock_variants(pam_variants, path_to_scores, path_to_pdbs='', complex_docking
                         help='path to output directory for variant scores')
     parser.add_argument('--pdb_dir', metavar='D', default='', type=str,
                         help='path to root of variant pdb directories (default: "")')
+    parser.add_argument('--set_partners', metavar='S', default="B_ACD", type=str,
+                        help='string for docking partners (default: "B_ACD")')
+    parser.add_argument('--setup_foldtree', metavar='S', type=str,
+                        help='string for foldtree (default: None - use default ft)')
     parser.add_argument('--complex', metavar='B', nargs='?', const=True, default=False,
-                        type=str, help='[switch] select complex docking (default: "False")')
+                        type=bool, help='[switch] select complex docking (default: "False")')
```

Comment:
Yeah I agree. I was wondering if you had to run it as `python script.py --complex=True` or `python script.py --complex=False` and if it would work. But you only need to run it like `python script.py --complex` for it to do the complex docking, which is what I was thinking. So it looks good